src/locales/de/translations.json

```
@@ -93,29 +93,46 @@
     }
   },
   "services": {
-    "title": "Services",
-    "header": {
-      "title": "Development & Design",
-      "kicker": "Unsere Services",
-      "text": "Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements im Konzerngeschäft.\nNeuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär. Gern unterstützen wir auch Sie in den folgenden Disziplinen:"
+    "hero": "Leistungen",
+    "title": "Unser Angebot an Sie",
+    "kicker": "Plattformen, Produkt & Beratung",
+    "text": "Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements von Mittelstand bis Konzerne. Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär. Gern unterstützen wir auch Sie in den folgenden Disziplinen.",
```

I am not sure but this sounds a little bit weird.
What do you think about that?
```suggestion
    "text": "Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements von Mittelstand bis Konzernebene. Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär. Gern unterstützen wir auch Sie in den folgenden Disziplinen.",
```

---

thanks I merged your suggestion 👍